SUPPORT.md

```diff
@@ -0,0 +1,14 @@
+# Support
+
+## How to file issues and get help
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the [existing issues][github-issues] before filing new issues to avoid duplicates. For new issues, file your bug or feature request as a new Issue.
+
+For help and questions about using this project, please open a new GitHub issue or visit the [#aks-engine-users][aks-engine-users-slack] Kubernetes Slack channel. The AKS Engine project maintainers will respond to the best of their abilities.
```

Comment:
The please open a new GitHub issue seems redundant.
Maybe update this section to:

For help and questions about using the project please
- Checkout out our [FAQ](https://github.com/Azure/aks-engine/blob/master/docs/faq.md)
- Review our [Documentation](https://github.com/Azure/aks-engine/blob/master/docs/README.md)
- Reach out on [aks-engine-users-slack](https://kubernetes.slack.com/archives/CU3N85WJK)

Reply:
👍 Agreed, much better.